recipes/mlem/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "mlem" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9ab945158ca5ecd772744a50d6d8d871cc7e11e140319ea8ed93a5adc17bf5e9
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - tpi = tpi.cli:cli
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - dill
+    - requests
+    - isort
+    - docker
+    - pydantic
+    - click
+    - aiohttp
+    - aiohttp_swagger
+    - Jinja2
+    - fsspec
+    - fusepy
+    - cached-property
+    - entrypoints
+    - setuptools
+
+test:
+  imports:
+    - mlem
+    - mlem.api
+  commands:
+    - mlem -h
+
+about:
+  home: https://github.com/iterative/mlem
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENCE
+  summary: Version and deploy your ML models following GitOps principles
+
+  description: |
+    MLEM is a tool to help you version and deploy your Machine Learning models:
+    * Serialise any model trained in Python into ready-to-deploy format
+    * Model lifecycle management using Git and GitOps principles
+    * Provider-agnostic deployment
+
+extra:
+  recipe-maintainers:
+    - aguschin
+    - mike0sv
+    - shcheklein
```

fine with being added as conda-forge co-maintiner

```suggestion
    - aguschin
    - casperdcl
    - mike0sv
    - shcheklein
```